mvlearn/compose/wrap.py

```diff
@@ -0,0 +1,278 @@
+        else:
+            Xs, y = check_Xs_y(Xs, y)
+        self.n_views_ = len(Xs)
+        if type(self.base_estimator) is list:
+            if len(self.base_estimator) != self.n_views_:
+                raise ValueError(
+                    "The length of the estimators should be the same as the"
+                    "number of views"
+                )
+            self.estimators_ = self.base_estimator
+        else:
+            self.estimators_ = [
+                clone(self.base_estimator) for _ in range(self.n_views_)
+            ]
+        return self
+
+    def fit(self, Xs, y=None):
+        r"""Fit each estimator to the data.
+
+        Parameters
+        ----------
+        Xs : list of array-likes or numpy.ndarray
+             - Xs length: n_views
+             - Xs[i] shape: (n_samples, n_features_i)
+            The data to fit to.
+
+        y : array-like of length (n_samples,), optional (default None)
+            Targets for a supervised estimation task
+
+        Returns
+        -------
+        self : object
+            Returns the instance itself.
+        """
+        self._prefit(Xs, y)
+        for estimator, X in zip(self.estimators_, Xs):
+            estimator.fit(X, y)
+        return self
+
+
+class ViewClassifier(BaseWrapper):
+    r"""Apply a sklearn classifier to each view of a dataset
+
+    Build a classifier from multiview data by
+    using individual scikit-learn classifiers on each view.
+
+    Parameters
+    ----------
+    base_estimator : a sklearn classifier instance, or a list
+        Either a single sklearn classifier that will be applied to each
+        view. One clone of the estimator will correspond to each view.
+        Otherwise, it should be a list of estimators, of length the number of
+        views in the multiview dataset.
+
+    Attributes
+    ----------
+    n_views_ : int
+        The number of views in the input dataset
+
+    estimators_ : list of objects of length n_views_
+        The list of classifiers used to predict data labels. If
+        self.base_estimator is a single estimator, this is a list containing
+        clones of that estimator, otherwise it is one view of
+        self.base_estimator.
+
+    Examples
+    --------
+    >>> from mvlearn.datasets import load_UCImultifeature
+    >>> from mvlearn.compose import ViewClassifier
+    >>> from sklearn.linear_model import LogisticRegression
+    >>> Xs, y = load_UCImultifeature()
+    >>> clfs = ViewClassifier(LogisticRegression())
+    >>> y_hat = clfs.fit(Xs, y).predict(Xs)
+    >>> print(y.shape)
+    (2000,)
+    """
+    def predict(self, Xs):
+        """
+        Return the predicted class labels using majority vote of the
+        predictions from each view.
+
+        Parameters
+        ----------
+        Xs : list of array-likes or numpy.ndarray
+             - Xs length: n_views
+             - Xs[i] shape: (n_samples, n_features_i)
+            The data to predict
```

Comment:
There are lots of places in these functions where Xs bullet points have indentation such that they don't render very well on the docs.

https://190-206838300-gh.circle-artifacts.com/0/dev/references/compose.html#viewclassifier

This is probably how we should try to do it from now on: (taken from SimpleSplitter, which renders well)

        Xs : list of numpy.ndarray
            - Xs length: n_views
            - Xs[i] shape: (n_samples, n_features_i)
            The input multiview dataset

Which results in clean bullet points for its inverse_transform() https://190-206838300-gh.circle-artifacts.com/0/dev/references/compose.html#simplesplitter

This is probably something we can revisit for the documentation as a whole later, because there are definitely lots of places where the change from the old rendering to the new one has messed up the cleanliness of the formatting, but for now let's just change the ones on this PR at least.